cdi/embedded/src/test/java/org/infinispan/cdi/embedded/test/distexec/DistributedExecutorCDITest.java

```diff
@@ -89,11 +83,7 @@ public void run() {
       }
    }
 
-   static class ImpliedInputCacheCallable implements Callable<Integer>, Serializable {
-
-
-      /** The serialVersionUID */
-      private static final long serialVersionUID = 5770069398989111268L;
+   static class ImpliedInputCacheCallable implements Callable<Integer>, Serializable, ExternalPojo {
```

Comment:
@slaskawi There should be no consequence for user classes, `ExternalPojo` is only needed for our test classes (see `ExternallyMarshallable.isAllowed()` and `ExternallyMarshallable.isMarshallablePackage()`).